Dockerfile

```
@@ -0,0 +1,16 @@
+FROM ubuntu:22.04 as ubuntu
+
+WORKDIR /root
+
+COPY requirements.txt ./requirements.txt
+COPY constants.py constants.py
+COPY ingest.py ingest.py
+COPY run_localGPT.py run_localGPT.py
+COPY utils.py utils.py
+COPY SOURCE_DOCUMENTS/ SOURCE_DOCUMENTS/
```

Is it not better to have something like 
```suggestion
COPY . .
`````

And have a `.dockerignore` to ignore what is not needed? This is probably more maintainable you may be missing some important files that will be added later in the project if someone adds them but forgot to update the dockerfile.